tests/mock_network/utils.rs

```diff
@@ -629,12 +636,66 @@ fn add_nodes_to_subsections_and_poll(
     poll_until_last_nodes_joined(env, nodes, first_index);
 }
 
+fn remove_elders_from_section_and_poll(
+    env: &Environment,
+    nodes: &mut Vec<TestNode>,
+    prefix: &Prefix<XorName>,
+    count: usize,
+) {
+    for _ in 0..count {
+        let removed_id = remove_elder_from_section(nodes, prefix);
+        poll_until(env, nodes, |nodes| node_left(nodes, &removed_id));
+        update_neighbours_and_poll(env, nodes, 2);
+    }
+}
+
 fn poll_until_last_nodes_joined(env: &Environment, nodes: &mut [TestNode], first_index: usize) {
     poll_until(env, nodes, |nodes| {
         all_nodes_joined(nodes, first_index..nodes.len())
     })
 }
 
+// Poll until the section with `prefix` has at least 4 elders which is the miminum so that if one
+// elder is removed, the consensus on it can still be reached.
+fn poll_until_minimal_elder_count(
+    env: &Environment,
+    nodes: &mut [TestNode],
+    prefix: &Prefix<XorName>,
+) {
+    let expected = 4;
```

Comment:
why 4?
better not to use a magic number here

Reply:
It's explained in the comment of this function. But maybe better to use a constant?

Reply:
ah, sry, I missed the comment.
And, yes, will prefer it to be a constant